### PR TITLE
T1890

### PR DESCRIPTION
**Pull request by secretrobotron**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/4abc41fd2d7dff8/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/4abc41fd2d7dff8/output.txt

Total script time: 0.91 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/807c58ee2302192/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/807c58ee2302192/output.txt

Total script time: 0.62 mins

Server is running on http://pr711.butyr.org/templates/test.html

File: editors/googlemap-editor.js [hunk 1]
```diff
@@ -232,8 +237,12 @@
           getMapFromTrackEvent();
         });
         setup( trackEvent );
+        _extraStyleTag = compiledLayout.querySelector( 'style' );
```
Comment (mjschranz) [inline editors/googlemap-editor.js]: Double Quotes

File: src/editor/base-editor.js [hunk 2]
```diff
@@ -59,6 +64,32 @@ define( [ "core/eventmanager" ], function( EventManagerWrapper ) {
...
 
+    /**
+     * Member: applyExtraStyleTag
+     *
+     * If a style tag is present in the given layout, place it in the document's head.
+     *
+     * @param {DOMFragment} layout: DOMFragment containing the style tag
+     */
+    extendObject.applyExtraStyleTag = function( layout ) {
+      _extraStyleTag = layout.querySelector( 'style' );
```
Comment (mjschranz) [inline src/editor/base-editor.js]: double quotes around style please.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/fa9d31187421e10/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/fa9d31187421e10/output.txt

Total script time: 0.80 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/3bdd0cffa611ac1/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/3bdd0cffa611ac1/output.txt

Total script time: 1.20 mins

Server is running on http://pr711.butyr.org/templates/test.html

File: src/editor/base-editor.js (hunk 2 above)
Comment (secretrobotron) [inline src/editor/base-editor.js]: Hahaha wow. Good catch.

File: src/editor/trackevent-editor.js [hunk 3]
```diff
@@ -30,10 +32,25 @@ define( [ "util/lang", "util/keys", "./base-editor",
...
+      if ( oldOpenEvent ) {
+        oldOpenEvent.apply( this, arguments );
+      }
+    };
+
     BaseEditor( extendObject, butter, rootElement, events );
 
     extendObject.defaultLayouts = __defaultLayouts.cloneNode( true );
 
+    // A horizontal scrollbar may be added if lots of editor content is available (perhaps a large manifest is present for a plugin).
```
Comment (dseif) [inline src/editor/trackevent-editor.js]: Should this be `// A vertical scrollbar may be added....`?

Comment (secretrobotron) [inline src/editor/trackevent-editor.js]: Indeed! Thank you.

**Pull request merged**